src/conn/conn_api.c

```diff
@@ -1048,6 +1048,69 @@ __conn_load_extensions(WT_SESSION_IMPL *session, const char *cfg[], bool early_l
     return (ret);
 }
 
+/*
+ * __conn_add_target_uri --
+ *     Add the target uri to the backup uri hash table.
+ */
+static int
+__conn_add_target_uri(WT_SESSION_IMPL *session, const char *name, size_t len)
```

Comment:
The conn_api.c file is already big - could the new code mostly live in a different file?

I'd suggest using __conn_add_backup_target_uri - since I think it's better if the name clearly indicates what it's being saved for.

Reply:
I have placed this in metadata file now, let me know what you think.